### ISSUE TITLE
Energy shields sometimes aren't working

### ISSUE THREAD

**Issue by Omegatron9** [lehjr/MachineMusePowersuits#66]
**Version info**
Java version: 1.8.0_301
Minecraft Forge version: 36.2.23
Minecraft version: 1.16.5
Mod version: 2.2.31

**Describe the bug**
Energy shield modules sometimes aren't providing any protection. I have batteries installed and the shield strength set to maximum. The tinker table GUI says it is providing armour but my armour bar and the tooltip on the item show that it isn't.

Usually the first energy shield installed will work properly, but then energy shields installed in the other pieces of the armour won't. Sometimes, when I take the energy shield out of the first piece and put it back in it stops working too, but it's not consistent.

Conventional plating seems to work fine.

**Expected behavior**
I should be able to install energy shields on all four pieces of the armour, and this protection should show up on my armour bar and the tooltip.

**Mod list**
[modlist.txt](https://github.com/lehjr/MachineMusePowersuits/files/10473020/modlist.txt)

**Comment by eyeonus** [comment]
The shield modules get disabled if the suit runs out of power and must be manually turned back on by pressing the key assigned to toggle the module. From my own playing, this seems like the only module that has this behaviour, and it is super annoying.

**Comment by Omegatron9** [comment]
I don't think that's the problem I'm having, I'm not running out of power.

**Comment by eyeonus** [comment]
Maybe not, but at the least I'm sure it's a related issue. When you are not getting armor from the shields, have you checked to see if the module is enabled?

**Comment by Omegatron9** [comment]
When this happens usually one piece of the armour provides protection but not the others. I don't think you can toggle off a module that's in multiple pieces of the armour on only one piece.

**Comment by eyeonus** [comment]
No, all energy shields are tied to the same toggle. The only other thing I can think of is to check the shields aren't at 0 on the shield strength slider in the tinker options, but you've probably already done that.

**Comment by Omegatron9** [comment]
Yeah, as I mentioned in the original issue I made sure the shield strength was at maximum and it didn't help.

**Comment by eyeonus** [comment]
Sorry, there's nothing in the OP that indicates your awareness that each shield module must be individually maxed. My mistake.
